packages/fxa-content-server/app/scripts/lib/fxa-client.js

```diff
@@ -1431,6 +1431,14 @@ FxaClientWrapper.prototype = {
       return getUpdatedSessionData(email, relier, accountData);
     });
   }),
+
+  verifyAccountThirdParty: withClient((client, relier, token, provider) => {
```

Comment:
nit: We've always been comically inconsistent between "login" and "signin", but I think one of those would be better than "verify" in names throughout here. Maybe also in the route name `/linked_account/login`